### ISSUE TITLE
npm run command can't find file?

### ISSUE THREAD

**Issue by f0rr0** [yarnpkg/yarn#782]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
```
yarn run start
yarn run v0.15.1
$ "$(npm bin)/babel-node scripts/start.js"
sh: /Users/...path.../node_modules/.bin/babel-node scripts/start.js: No such file or directory
error Command failed with exit code 127.
```
**If the current behavior is a bug, please provide the steps to reproduce.**
```
"scripts": {
"start" : "$(npm bin)/babel-node scripts/start.js"
}
```

**What is the expected behavior?**
The script should be *found* since it exists. `npm run start` handles this as expected.

**Please mention your node.js, yarn and operating system version.**
Node - v6.7.0
yarn - v0.15.1
OS - osx 10.10.5

**Comment by FLGMwt** [comment]
I think you'll be fine without the `$(npm bin)/` and just `"start": "babel-node scripts/start.js"` since `yarn run` includes the .bin of the working project (https://github.com/yarnpkg/yarn/blob/master/src/cli/commands/run.js#L35)

Additionally, the idea is to use the npm cli *or* yarn as they're serving the same purpose : )

**Comment by f0rr0** [comment]
@FLGMwt I get the same error after removing bin

**Comment by nodesocket** [comment]
I confirm as well. A direct port of `npm` to `yarn` breaks:

````json
"scripts": {
    "dev": "webpack-dev-server --progress --colors --port 9090"
}
````

`yarn run dev` errors with:

````bash
No such file or directory
error Command failed with exit code 127.
````

**Comment by notahat** [comment]
See https://github.com/yarnpkg/yarn/issues/733. I think this is the same problem.

**Comment by f0rr0** [comment]
`"start": "pwd"` works. But `"start": "echo 'stuff'"` fails. This validates https://github.com/yarnpkg/yarn/issues/733#issuecomment-253086168

**Comment by bobbo** [comment]
This is fixed in current master branch (see #809)

**Comment by kittens** [comment]
Fixed via #809.
